gisnav:
  mock_gps_node:
    ros__parameters:
      use_sensor_gps: True

